### PR TITLE
Python3 qt5 bindings

### PR DESCRIPTION
**Pull request by brawner**
This adds a new key for python3 compatible qt5 bindings "python3-qt5-bindings". In order to target python3 for ROS2, a new key needs to be added. Unfortunately, the Python3 compatible PySide2 is not yet available in ubuntu bionic/xenial yet so that has been removed. I think they are targeting Ubuntu cosmic for that package.

### CODE REVIEW

File: rosdep/sources.list.d/19-default.list [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+# generic
+yaml https://raw.githubusercontent.com/brawner/rosdistro/master/rosdep/base.yaml
+yaml https://raw.githubusercontent.com/brawner/rosdistro/master/rosdep/python.yaml
+yaml https://raw.githubusercontent.com/brawner/rosdistro/master/rosdep/ruby.yaml
+gbpdistro https://raw.githubusercontent.com/brawner/rosdistro/master/rosdep/fuerte.yaml fuerte
+
+https://raw.githubusercontent.com/brawner/rosdistro/master/rosdep/osx-homebrew.yaml
+
+# newer distributions (Groovy, Hydro, ...) must not be listed anymore, they are being fetched from the rosdistro index.yaml instead
```
[the remark this answers is not in the document]
Comment (brawner) [reply rosdep/sources.list.d/19-default.list]: I was following http://docs.ros.org/independent/api/rosdep/html/contributing_rules.html

I guess it's not needed for testing then?

File: rosdep/python.yaml [hunk 2]
```diff
@@ -4593,6 +4593,23 @@ python3-pytest:
...
+  gentoo: [dev-python/PyQt5]
+  opensuse: [python3-qt5]
+  slackware: [python3-PyQt5]
+  ubuntu:
+    artful: [pyqt5-dev, python3-pyqt5, python3-pyqt5.qtsvg, python3-sip-dev]
+    bionic: [pyqt5-dev, python3-pyqt5, python3-pyqt5.qtsvg, python3-sip-dev]
+    wily: [pyqt5-dev, python3-pyqt5, python3-pyqt5.qtsvg, python3-sip-dev]
+    xenial: [pyqt5-dev, python3-pyqt5, python3-pyqt5.qtsvg, python3-sip-dev]
+    yakkety: [pyqt5-dev, python3-pyqt5, python3-pyqt5.qtsvg, python3-sip-dev]
+    zesty: [pyqt5-dev, python3-pyqt5, python3-pyqt5.qtsvg, python3-sip-dev]
```
[the remark this answers is not in the document]
Comment (brawner) [reply rosdep/python.yaml]: Removed individual distros and consolidated for debian and ubunt. I also moved this to the python3 section lower down

File: rosdep/sources.list.d/19-default.list (hunk 1 above)
[the remark this answers is not in the document]
Comment (dirk-thomas) [reply rosdep/sources.list.d/19-default.list]: That part of the instructions is only for testing the changes locally.

[the remark this answers is not in the document]
Comment (brawner) [reply rosdep/sources.list.d/19-default.list]: Ok, it's removed

**Review by dirk-thomas — APPROVED** [review]

**Review by clalancette — APPROVED** [review]

**Pull request merged**